src/roles/foreman/defaults/main.yaml

```diff
@@ -19,27 +19,19 @@ foreman_puma_workers: "{{ [32, ansible_facts['processor_nproc'] * 1.5, (ansible_
 
 foreman_recurring_tasks_enabled: true
 foreman_recurring_tasks:
-  - instance: reports-daily
-    rake: "reports:daily"
+  - instance: hourly
+    rake: "cron:hourly"
+    schedule: "hourly"
+  - instance: daily
+    rake: "cron:daily"
     schedule: "daily"
-  - instance: db-sessions-clear
-    rake: "db:sessions:clear"
-    schedule: "daily"
-  - instance: reports-expire
-    rake: "reports:expire"
-    schedule: "daily"
-  - instance: audits-expire
-    rake: "audits:expire"
-    schedule: "daily"
-  - instance: reports-weekly
-    rake: "reports:weekly"
+  - instance: weekly
+    rake: "cron:weekly"
     schedule: "weekly"
-  - instance: reports-monthly
-    rake: "reports:monthly"
+  - instance: monthly
+    rake: "cron:monthly"
     schedule: "monthly"
-  - instance: notifications-clean
-    rake: "notifications:clean"
-    schedule: "weekly"
+
   - instance: ldap-refresh_usergroups
     rake: "ldap:refresh_usergroups"
     schedule: "*-*-* *:00,30:00"
```

Comment:
would it be sufficient to run this `hourly` too?

:eyes: @adamruzicka

Reply:
I think so, yes.